OlinkAnalyze/R/Olink_anova.R

```diff
@@ -37,7 +37,7 @@
 #'
 #' library(dplyr)
 #'
-#' npx_df <- npx_data1 %>% dplyr::filter(!grepl('control',SampleID, ignore.case = TRUE))
+#' npx_df <- npx_data1 %>% filter(!grepl('control',SampleID, ignore.case = TRUE))
```

Comment:
should we selectively import in examples?

Reply:
No, we agreed on using library(dplyr), that's why it was removed. The dplyr::filter call is currently in the file in develop and shouldn't be there.